config/environments/test.rb

```diff
@@ -28,7 +28,7 @@
   config.assets.digest = ENV.key?('RAILS_DISABLE_ASSET_DIGEST') ? false : true
 
   config.middleware.use RackSessionAccess::Middleware
-  config.lograge.enabled = true
+  config.lograge.enabled = false
```

Comment:
Not sure if this is an OK change to make?

Reply:
I think it was just enabled for terseness

I think that if we're going to make a script that depends on it, we should either
- leave a comment describing that it needs to be off
- check it based on some environment variable that corresponds to the rake task for aXe

Reply:
Yeah, that's a good idea. aXe tests are just part of the specs.  I'll change it to switch based on `ENV['COVERAGE']` and document that :+1: